README.md

```diff
@@ -110,6 +110,35 @@ The image will be rendered inside the displayed dataframe and next to the "gener
 
 (*Note: you also need an `OPENAI_API_KEY` environment variable to use `gpt-4-vision-preview`*)
 
+## Using input data in prompts
+
+The user/system prompt textboxes support [Jinja](https://jinja.palletsprojects.com/) templates. 
+Given a column named `text` in your uploaded CSV data, you can use values from this column by 
+writing the simple `{{text}}` template in your prompt.
+
+If the values in your column represent more complex objects such as Python dictionaries or lists,
+you can still work with them but make sure that these values are enclosed in double quotes (`"`) 
+in your CSV file. E.g. given a column `texts` with a value like `"[\"A\", \"B\", \"C\"]'`, you can 
+utilise this template to enumerate the individual list items in your prompt:
```

Comment:
They have to be escaped with `"`.  Single quoting would work only if we specified the non-default `quotechar="'"` inside Prompterator in `pd.read_csv` but then double quoting wouldn't work, and I thought that supporting the default quotechar (`"`) is better.

As for JSON objects: In the ideal world, everything would be a valid JSON string and all would work nicely. However, requiring valid JSON strings would cause a lot of pain to users. If you create your CSV file by saving a dataframe with `df.to_csv`, you'll end up with JSON-invalid strings like this:
```csv
col3,col4
{'a': 'b'},"['a', 'b']"
{'c': 'd'},"['c', 'd']"
```
I couldn't even find the `pd.to_csv` args combination that would lead to double-quoted values that are still valid JSON strings (i.e. use double quotes also internally). Maybe I didn't try hard enough but one thing is for sure -- requiring everyone to undergo this when producing their CSV file sounds like too much.

Now giving it a second thought, maybe what I just wrote is an argument in favour of removing `fromjson` altogether or making it use `ast.literal_eval` under the hood. I guess I'd keep `fromjson` (since it seems to be a widespread name that people may be used to with Jinja). Using `ast.literal_eval` inside `fromjson` would mean accepting non-JSON stuff in `fromjson` which smells but maybe it would be the simplest? Wdyt @sweco?

Reply:
I suggest we discuss this on a quick huddle but this is what I normally do to obtain a valid JSON in a CSV file.

```python
df = pd.DataFrame.from_records([{"c1": ["a", "b"], "c2": {"a": "b"}}])
df["c1"] = df["c1"].apply(json.dumps)
df["c2"] = df["c2"].apply(json.dumps)
df.to_csv("test.csv", index=False)

# c1,c2
# "[""a"", ""b""]","{""a"": ""b""}"
```

The reason I'm bringing this up is that I find it a good idea to simply stick to the standards. By saying that it should be a valid CSV file with a valid JSON in the column, we make the description clearer (at least for me) and simplify it as well - we don't have to mention all the specifics of the CSV and JSON formats, e.g. that quotes should be double-escaped, that if there is a comma, it should also be enclosed in the quotes, etc.

However, I get that the resulting CSV is not very nice, so let's discuss this. 😊